app/src/main-process/menu/build-default-menu.ts

```diff
@@ -132,10 +132,10 @@ export function buildDefaultMenu(
         click: emit('create-commit'),
       },
       {
-        label: __DARWIN__ ? 'Compare to Branch' : 'C&ompare',
-        id: 'compare-to-branch',
+        label: __DARWIN__ ? 'History' : 'Hist&ory',
```

Comment:
Two suggestions:
 - I think we prefix this with a verb, like `View History` or `Show History`, for macOS.
 - For non-macOS we originally had `&H` as the keyboard shortcut here (see 24f20606d700fd216bc3c0666342f0222cf6fab1) so I think we can put that back here without having a conflict.

Reply:
> IMO, I feel that this menu item belongs under the Repository menu since we want to show history for the repository.

For the moment let's focus just on the name of the menu item. We've been using the View menu as a gathering point to collect all the options for accessing parts of the main user interface, but we can have a discussion about whether that still makes sense later.